drivers/bluetooth/hci/h4.c

```diff
@@ -434,6 +437,10 @@ static int h4_open(void)
 	if (nrf51_init(h4_dev) < 0) {
 		return -EIO;
 	}
+#elif defined(CONFIG_BT_NRF52840_PCA10090_GPIO_RESET)
```

Comment:
I would be OK with letting this one go through, but I really want @jhedberg to weigh in.
The alternative is to have a generic hook such as:

```
#if defined(CONFIG_BT_DRIVER_OPEN_HOOK)
    if (bt_driver_open_hook(h4_dev) < 0) {
        return -EIO;
    }
#endif
```

And then have the implementation of `bt_driver_open_hook()` in the `board.c` of the corresponding board.